Source/WebKit/Platform/SharedMemory.h

```diff
@@ -151,4 +119,50 @@ class SharedMemory : public ThreadSafeRefCounted<SharedMemory> {
 #endif
 };
 
+class SharedMemoryHandle {
```

Comment:
What’s the rationale for moving the `SharedMemoryHandle` class down within the header? It makes this patch harder to review, so maybe leave it out?

Reply:
Will fix

Reply:
Moved back to its original position